topic2_4/target/tomcat/conf/web.xml

```diff
@@ -0,0 +1,1200 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files